frontend/tailwind.config.cjs

```diff
@@ -293,6 +295,37 @@ const config = {
 					color: 'transparent',
 					backgroundClip: 'text',
 					backgroundImage: `linear-gradient(to right, ${theme('colors.blue.600')}, ${theme('colors.blue.500')})`
+				},
+				'.splitpanes__pane': {
+					backgroundColor: theme('colors.white') + ' !important',
+					overflow: 'auto !important',
+				},
+				'.splitpanes__splitter': {
+					backgroundColor: theme('colors.gray.300') + ' !important',
+					margin: '0 !important',
+					border: 'none !important',
+					'&::before': {
+						backgroundColor: '#00000060 !important',
+					},
+					'&::after': {
+						content: 'none !important'
+					},
+				},
+				'.splitpanes--vertical>.splitpanes__splitter': {
+					width: '5px !important',
```

Comment:
5px seems too big imho. I think we can set it to 3, set the handle to 1, and leave 1px on each side.
<img width="64" alt="Screenshot 2022-10-15 at 23 42 27" src="https://user-images.githubusercontent.com/456655/196008644-9830f9bd-9e37-426e-baa1-11ab05d2164f.png">

Reply:
Yep, the looks of the splitter is only temporary.